src/tests/ftest/util/soak_test_base.py

```diff
@@ -553,6 +554,7 @@ def run_soak(self, test_param):
         job_list = self.params.get("joblist", test_param + "*")
         resv_bytes = self.params.get("resv_bytes", test_param + "*", 500000000)
         ignore_soak_errors = self.params.get("ignore_soak_errors", test_param + "*", False)
+        self.enable_il = self.params.get("enable_intercept_lib", test_param + "*", False)
```

Comment:
Where are we setting enable_intercept_lib to true?

Reply:
Correct it will be disabled until the feature branch is merged into master